build-tools/docker/example-mainnet/docker-compose.yml

```diff
@@ -0,0 +1,101 @@
+version: '3'
+
+services:
+  node-daemon:
+    image: $DOCKERHUB_USERNAME/node-daemon:$ML_SOFTWARE_VERSION
+    volumes:
+      - ./container_home:/home/mintlayer
```

Comment:
I would call it `./mintlayer-data`. We should use names that minimize the need to change.